bedrock/firefox/templates/firefox/os/index-new.html

```diff
@@ -37,82 +51,48 @@
-          </a>
-          <div class="stage scroll-homescreen" data-button="#scroll-homescreen">
-            <img src="{{ static('img/firefox/os-new/device.png') }}" width="304" height="637" alt="" />
-            <div class="homescreen"></div>
-          </div>
-
-          <div class="stage swipe-apps" data-button="#swipe-apps">
-            <img src="{{ static('img/firefox/os-new/device.png') }}" width="304" height="637" alt="cellphone outlines" />
-            <img src="{{ static('img/firefox/os-new/wireframes-sprite.png') }}" class="wireframe-screens" height="499" width="1205" alt="" />
-            <div class="screens-sprite"></div>
-          </div>
-
-          <ul>
-            <li><a href="#" id="scroll-homescreen" class="active-state">{{_('Scroll the homepage')}}</a></li>
-            <li><a href="#" id="swipe-apps">{{_('Swipe between apps')}}</a></li>
-          </ul>
-        </div> #}
       </div>
     </section>
     <section class="fxos-features">
       <div class="content-wrapper">
         <h2>{{_('Smart has never been so simple')}}</h2>
-        <div class="feature">
-          <img src="{{ static('img/firefox/os-new/icons/scroll_icon.png') }}" width="140" height="140" alt="{{_('interaction icon')}}" />
-          <h3>{{_('Scroll for what you need')}}</h3>
-          <p>{{_('Say goodbye to multiple home screens. Everything you need is right there in one clean, continuous scroll. What could be easier?')}}</p>
-        </div>
-
-        <div class="feature">
-          <img src="{{ static('img/firefox/os-new/icons/search_icon.png') }}" width="140" height="140" alt="{{_('interaction icon')}}" />
-          <h3>{{_('Search in a snap')}}</h3>
-          <p>{{_('Search the Web and the Firefox Marketplace, or visit a website directly —  all from the search bar at the top of your home screen.')}}</p>
-        </div>
-
-        <div class="feature">
-          <img src="{{ static('img/firefox/os-new/icons/swipe_icon.png') }}" width="140" height="140" alt="{{_('interaction icon')}}" />
-          <h3>{{_('Swipe&nbsp;between apps')}}</h3>
-          <p>{{_('Access your recently used apps in a flash. Just swipe from the edge of the screen to cycle through all of your open apps.')}}</p>
-        </div>
+        <ul class="features">
+          <li>
+            <div class="icon scroll"></div>
+            <h3>{{_('Scroll for what you need')}}</h3>
+            <p>{{_('Say goodbye to multiple home screens. Everything you need is right there in one clean, continuous scroll. What could be easier?')}}</p>
+          </li>
+          <li>
+            <div class="icon search"></div>
+            <h3>{{_('Search in a snap')}}</h3>
+            <p>{{_('Search the Web and the Firefox Marketplace, or visit a website directly —  all from the search bar at the top of your home screen.')}}</p>
+          </li>
+          <li>
+            <div class="icon swipe"></div>
+            <h3>{{_('Swipe&nbsp;between apps')}}</h3>
```

Comment:
It's actually from the original landing: it's a dirty trick to differentiate the string from "Swipe between apps". One is used as a title, the other as a button label, some locales need separate strings.